### PR TITLE
Hypo71 improvements

### PR DESCRIPTION
**Pull request by jmsaurel**
Correct a bug identified by Aleksey Emanov when renaming phases in scolv.

Adds a feature to map time.uncertainties and Hypo71 weights based on a static table (a given time.uncertainty will always map to the same Hypo71 weight value).

Update Hypo71 documentation.

### CODE REVIEW

**Comment by gempa-jabe** [comment]
I would rephrase the commits and prepend "[hypo71] ", see 5bca394d867b4798b6976d6835d5b89e7ce8c182. If you can't or don't want to do it, I can resolve it for you. Otherwise just run

```
git rebase -i origin/master
```

and replace **pick** with **reword** for each line. This assumes that *origin/master* is our master.

Thank you.

**Comment by gempa-jabe** [comment]
I have picked and reworded your commits. Thanks.

**Comment by jmsaurel** [comment]
Sorry Jan, I didn't had much time those latest days.
Thank for the rewording.

Jean-Marie.
